packages/storybook/constants/globalisation.js

```diff
@@ -1,4 +1,10 @@
+export const AUSTRALIAN_LOCALE = 'en-AU';
+export const DANISH_LOCALE = 'da-DK';
 export const ENGLISH_LOCALE = 'en-GB';
+export const IRISH_LOCALE = 'en-IE';
+export const ITALIAN_LOCALE = 'it-IT';
+export const NEW_ZEALAND_LOCALE = 'en-NZ';
+export const NORWEIGAN_LOCALE = 'nb-NO';
 export const SPANISH_LOCALE = 'es-ES';
 
 export const VALID_LOCALES = [ENGLISH_LOCALE, SPANISH_LOCALE];
```

Comment:
Only thing I was unsure of was if the constants added above should be included in the valid locales array?